llvm/test/CodeGen/AArch64/sve-extract-scalable-vector.ll

```diff
@@ -19,9 +206,6 @@ define <vscale x 1 x i16> @extract_nxv1i16_nxv6i16(<vscale x 6 x i16> %vec) noun
   ret <vscale x 1 x i16> %retval
 }
 
-declare <vscale x 1 x i32> @llvm.vector.extract.nxv1i32.nxv4i32(<vscale x 4 x i32>, i64)
-declare <vscale x 1 x i16> @llvm.vector.extract.nxv1i16.nxv6i16(<vscale x 6 x i16>, i64)
```

Comment:
We don't need to declare intrinsics anymore so rather than add more I figured I'd just remove the existing ones. I'm happy to re-add them if reviews prefer to keep them.